src/pages/About.jsx

```diff
@@ -1,11 +1,74 @@
+import React, { useEffect, useState } from 'react';
+
+import ReactMarkdown from 'react-markdown';
+
 import ErrorBoundary from '../components/ErrorBoundary.jsx';
 
+x;
 function About() {
+  const [content, setContent] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  /**
+   * Fetches and loads the about us page markdown content /md file
+   * ReactMarkdown is needed  to be installed to render the markdown content
+   * npm install react-markdown
+   * The content should be styled accordingly .I have not done that.
+   * The header ,sub headers  and blockquote are not shown as they should be .
+   * npm install @tailwindcss/typography --save-dev
+   * this plug in is needed to be installed to style the markdown content
+   * The plugin should be added to the tailwind.config.js file or similar I did not want to add even more files and consfigs for now .just wanted to fetch the content and render it, and be a ble to edit via cms
+   */
+  useEffect(() => {
+    const fetchMarkdown = async () => {
```

Comment:
@AngelosZar Great job! Thanks for putting in the effort. However, the data is currently fetched as a string. We still need a function to convert that string into an object. I recommend using an npm package called [front-matter](https://www.npmjs.com/package/front-matter). It helps extract the metadata directly from the Markdown file into an object. In this case, you won't need react-markdown.